src/modules/ekf2/EKF/ekf.h

```diff
@@ -121,8 +121,8 @@ class Ekf final : public EstimatorInterface
 #if defined(CONFIG_EKF2_OPTICAL_FLOW)
 	const auto &aid_src_optical_flow() const { return _aid_src_optical_flow; }
 
-	const Vector2f &getFlowVelBody() const { return _flow_vel_body; }
-	const Vector2f &getFlowVelNE() const { return _flow_vel_ne; }
+	const Vector2f &getFlowVelBody() const { return _flow_vel_body.getState(); }
```

Comment:
it would be nice to still log the "raw" values

Reply:
updated to keep both